libs/category/state/src/facades/category.facade.ts

```diff
@@ -83,11 +83,11 @@ export class DaffCategoryFacade<
 	}
 
 	getProductsByCategory(categoryId: V['id']): Observable<W[]> {
-	  return this.store.pipe(select(this.categorySelectors.selectProductsByCategory, { id: categoryId }));
+	  return this.store.pipe(select(this.categorySelectors.selectProductsByCategory(categoryId)));
```

Comment:
Is this sufficient to memoize each of the selector instances? Wouldn't it create a new selector each time this method is called? Wouldn't we need to cache each of the created selectors per-prop?